smoke-test/src/test/java/org/opennms/e2e/containers/util/ssh/SshClient.java

```diff
@@ -0,0 +1,214 @@
+/*******************************************************************************
```

Comment:
Can we move the utility classes into `src/main/java` instead of `src/test/java`?